File: verve patch/base/Applications/CSymphony_0_1/csSamples/CodeGotoStatement.cs
using System;
using System.CodeDom;
using System.CodeDom.Compiler;
using System.Collections;
using CSymphony.Lib;

namespace CSymphony.CSy
{
	public class CSyUnit : GraphUnit
	{
		public override void Build()
		{
			ns("MyNameSpace");

			declClass(TypePublic, "Type1");
			declMethod(MemPublic, "System.Void", "MonMethod");

			jumpTo("JumpToLabel");

			MethodRef w = new MethodRef ("System.Console", "WriteLine");

			enter (ex (w , new Literal ("Test Output")));

			declLabel("JumpToLabel", ex (w, new Literal ("Output from Labeled statement")));

		}
	}
}

File: verve patch/base/Applications/CSymphony_0_1/vbSamples/CodeGotoStatement.vb
Option Strict Off
Option Explicit On

Imports System
Imports System.CodeDom
Imports System.CodeDom.Compiler
Imports System.Collections
Imports CSymphony.Lib

Namespace CSymphony.CSy
	Public Class CSyUnit
		 Inherits GraphUnit
		Overrides Public Sub Build()
			ns("MyNameSpace")

			declClass(TypePublic, "Type1")
			declMethod(MemPublic, "System.Void", "MonMethod")

			jumpTo("JumpToLabel")

			Dim w As MethodRef  = New MethodRef ("System.Console", "WriteLine")

			enter (ex (w , newLit("Test Output")))

			declLabel("JumpToLabel", ex (w, newLit("Output from Labeled statement")))
		End Sub
	End Class
End Namespace
			